Radzen.Blazor/RadzenYearView.razor.cs

```diff
@@ -0,0 +1,119 @@
+using Microsoft.AspNetCore.Components;
+using Radzen.Blazor.Rendering;
+using System;
+using System.Drawing;
+using System.Globalization;
+
+namespace Radzen.Blazor
+{
+    /// <summary>
+    /// Displays the appointments in a month day in <see cref="RadzenScheduler{TItem}" />
+    /// </summary>
+    /// <example>
+    /// <code>
+    /// &lt;RadzenScheduler Data="@appointments"&gt;
+    ///     &lt;RadzenMonthView /&gt;
+    /// &lt;/RadzenScheduler&gt;
+    /// </code>
+    /// </example>
+    public partial class RadzenYearView : SchedulerViewBase
+    {
+        /// <inheritdoc />
+        public override string Icon => "view_list";
+
+        /// <inheritdoc />
+        public override string Title
+        {
+            get => Scheduler.CurrentDate.ToString("yyyy", Scheduler.Culture);
+        }
+
+        /// <inheritdoc />
+        [Parameter]
+        public override string Text { get; set; } = "Year";
+
+        /// <summary>
+        /// Specifies the maximum appointnments to render in a slot.
+        /// </summary>
+        /// <value>The maximum appointments in slot.</value>
+        [Parameter]
+        public int? MaxAppointmentsInSlot { get; set; }
+
+        /// <summary>
+        /// Specifies the text displayed when there are more appointments in a slot than <see cref="MaxAppointmentsInSlot" />.
+        /// </summary>
+        /// <value>The more text. Set to <c>"+ {0} more"</c> by default.</value>
+        [Parameter]
+        public string MoreText { get; set; } = "+ {0} more";
+
+        /// <summary>
+        /// Specifies the background color of the Month label in the view />.
+        /// </summary>
+        /// <value>The color of the month labels. Will use current theme if not set.</value>
+        [Parameter]
+        public string MonthLabelColor { get; set; } = "";
+
+        /// <summary>
+        /// Specifies the background color of the cells that don't fall in the month for that row in the view />.
+        /// </summary>
+        /// <value>The color of the cells in a month row that are not part of the previous or next month. Will use current theme if not set.</value>
+        [Parameter]
+        public string NotInMonthColor { get; set; } = "";
```

Comment:
Same as `MonthLabelColor` - is there a need for this to be a property and not set through CSS?

Reply:
I've created scss and css variables for each of these in _scheduler.scss and removed them as properties.